languages/python/exercises/concept/comments/.docs/after.md

```diff
@@ -0,0 +1,38 @@
+
+There are two types of comments used in Python
+1. Single line comment.
+2. Multi-line comment.
+
+
+1. Single line comment 
+Single line comment is starts with '#'. This is only useful for only one line comment.
+Comment is only valid for current line.
+
+e.g.
+
+# This is single line comment 
```

Comment:
Because this file is in Markdown format, a `#` character creates a header, instead of a comment like it would in a Python file.

In order to show an example of Python code, you can use a [Markdown code block](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet), and then write the Python-formatted code inside of that. Then, the Python comments will render the way that is intended here.

Please also see the Style Guide for implementing Concept exercises; it has some more helpful links and notes about formatting.
https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md